var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -47,6 +47,8 @@ class Warpx(CMakePackage):
             description='Build the WarpX executable application')
     variant('ascent', default=False,
             description='Enable Ascent in situ vis')
+    variant('sensei', default=False,
+            description='Enable SENSEI in situ vis')
```

Comment:
I know I added this, let's write the full `visualization` for readability :-)
```suggestion
            description='Enable Ascent in situ visualization')
    variant('sensei', default=False,
            description='Enable SENSEI in situ visualization')
```